typegate/src/runtimes/prisma.ts

```diff
@@ -37,6 +42,59 @@ function isPrismaOperationMat(mat: Materializer): mat is PrismaOperationMat {
   return mat.name === "prisma_operation";
 }
 
+export interface PrismaRuntimeDS extends Omit<TGRuntime, "data"> {
+  data: PrismaRuntimeData;
```

Comment:
Omit, then re-add the data field, why?

Reply:
Their types were incompatible, TypeScript complained...